jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -430,7 +432,8 @@ private void handleColibriRequest(XmppConnection.ColibriRequest request)
             return createConference(
                     conferenceIq.getName(),
                     ColibriUtil.parseGid(conferenceIq.getGID()),
-                    conferenceIq.getMeetingId());
+                    conferenceIq.getMeetingId(),
+                    conferenceIq.isRtcStatsEnabled());
```

Comment:
This will result in rtcstats being disabled when the bridge is used with older jicofo versions or other colibri implementations. Can we default to `true` (just like we default to `true` for the flag in jicofo)

Reply:
Yes, I will make the appropriate change in jitsi-xmpp-extensions.